pkg/models/network.go

```diff
@@ -226,6 +226,20 @@ func (n *Network) BootstrappingContext() (context.Context, context.CancelFunc) {
 	return context.WithTimeout(context.Background(), timeout)
 }
 
+func (n Network) SDKNetwork() sdkNetwork.Network {
```

Comment:
i feel like we should deprecate CLI network in favour of SDK network so this bridge is not needed.
should be a separate PR for this

Reply:
yes, a separate PR is needed